src/styles/blocks/reset.scss

```diff
@@ -0,0 +1,21 @@
+* {
+  box-sizing: border-box;
+}
+  
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+}
+  
+body {
+  margin: 0;
+}
+  
+h2,
+h3,
+p {
+  margin: 0;
+  cursor: default;
```

Comment:
This si default value.

Reply:
sorry, but cursor is not default.  deafult value is cursor text for this elements

Reply:
okay, agreed. But why do you need to change it?

Reply:
because I see no reason to highlight the text in the card